lib/std/debug.zig

```diff
@@ -1695,7 +1721,7 @@ fn getDebugInfoAllocator() *mem.Allocator {
 pub const have_segfault_handling_support = switch (builtin.os.tag) {
     .linux, .netbsd => true,
     .windows => true,
-    .freebsd, .openbsd => @hasDecl(os, "ucontext_t"),
+    .freebsd, .openbsd, .haiku => @hasDecl(os, "ucontext_t"),
```

Comment:
Does Haiku support siginfo and has a definition for `ucontext_t`?

Reply:
Haiku has definitions for `ucontext_t` and from what i can tell has support for siginfo although i imagine a more seasoned haiku developer will have a more definitive answer

(see `/system/develop/headers/posix/signal.h` on a Haiku system)